compiler/rustc_middle/src/middle/codegen_fn_attrs.rs

```diff
@@ -1,7 +1,7 @@
 use crate::mir::mono::Linkage;
 use rustc_attr::{InlineAttr, InstructionSetAttr, OptimizeAttr};
 use rustc_span::symbol::Symbol;
-use rustc_target::spec::SanitizerSet;
+use rustc_target::{abi::Align, spec::SanitizerSet};
```

Comment:
We don't usually have this type of import merging in rustc.
```suggestion
use rustc_target::abi::Align;
use rustc_target::spec::SanitizerSet;
```